src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -112,10 +112,6 @@ def _read_hdf5_file(
         assert file_path is not None, "file_path must be specified when using stream_mode='remfile'"
         rfile = remfile.File(file_path)
         hdf5_file = h5py.File(rfile, "r")
-
```

Comment:
I don't think we need to remove this case as it works for the current file in the CI.

Reply:
I actually would like to avoid removing this, can we do that?